src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -143,31 +144,26 @@ private bool ValidateOptions(FileWorkItemsOptions options, SarifWorkItemContext
             return valid;
         }
 
-        private static bool ValidateHostUri(string hostUriString, SarifWorkItemContext workItemFilingConfiguration)
+        // Validates that the URI coming from either the commandline or configuration is valid for filing work items.  A null or empty 
+        // value is valid if and only if a plug-in will later provide the value.  This cannot be determined until plugins are loaded later,
+        // so at this point an empty value passes validation.
+        private static bool ValidateHostUri(string workItemUriString, SarifWorkItemContext workItemFilingConfiguration)
         {
-            if (string.IsNullOrEmpty(hostUriString) && workItemFilingConfiguration.HostUri == null)
-            {
-                // No project URI was provided via the --project-uri argument or as
-                // part of an input file specified via --configuration.
-                Console.Error.WriteLine(MultitoolResources.WorkItemFiling_NoHostUriSpecified);
-                return false;
-            }
 
-            Uri hostUri = null;
-            if (!string.IsNullOrEmpty(hostUriString) && !Uri.TryCreate(hostUriString, UriKind.RelativeOrAbsolute, out hostUri))
+            Uri workItemUri = null;
+            if (!string.IsNullOrEmpty(workItemUriString) && !Uri.TryCreate(workItemUriString, UriKind.RelativeOrAbsolute, out workItemUri))
```

Comment:
>RelativeOrAbsolute [](start = 102, length = 18)

I'm note sure how a relative URI makes sense. Shouldn't this just be absolute, since we're trying to extract the domain information?